### ISSUE TITLE
Creates quotation marks in Foundation 5.5.2 Magellan Class

### ISSUE THREAD

**Issue by devshack** [sass/node-sass#1397]
**MacOS X 10.11.3 / Node.js v5.6.0 / node-sass 3.4.2 / grunt-sass 1.1.0**

When using node-sass (via Grunt) with the SCSS of the Foundation Framework 5.5.2, the output creates quotation marks for this code:

```

@function data($attr) {

  @return '[data-' + $attr + ']';
}

$include-html-magellan-classes: 1 !default;

$magellan-bg: #ffffff !default;
$magellan-padding: 10px !default;


  @if $include-html-magellan-classes {

    #{data('magellan-expedition')}, #{data('magellan-expedition-clone')} {
      background: $magellan-bg;
      min-width: 100%;
      padding: $magellan-padding;
      z-index: 50;

      .sub-nav {
        margin-bottom: 0;
        dd { margin-bottom: 0; }
        a {
          line-height: 1.8em;
        }
      }
    }

  }

```
Expected result (works with ruby sass, too):
```
[data-magellan-expedition], [data-magellan-expedition-clone] {
  background: #ffffff;
  min-width: 100%;
  padding: 10px;
  z-index: 50; }
  [data-magellan-expedition] .sub-nav, [data-magellan-expedition-clone] .sub-nav {
    margin-bottom: 0; }
    [data-magellan-expedition] .sub-nav dd, [data-magellan-expedition-clone] .sub-nav dd {
      margin-bottom: 0; }
    [data-magellan-expedition] .sub-nav a, [data-magellan-expedition-clone] .sub-nav a {
      line-height: 1.8em; }
```
Result with node-sass (latest and 3.5.0 beta 1):
```
"[data-magellan-expedition]", [data-magellan-expedition-clone] {
  background: #ffffff;
  min-width: 100%;
  padding: 10px;
  z-index: 50; }
  "[data-magellan-expedition]" .sub-nav, [data-magellan-expedition-clone] .sub-nav {
    margin-bottom: 0; }
    "[data-magellan-expedition]" .sub-nav dd, [data-magellan-expedition-clone] .sub-nav dd {
      margin-bottom: 0; }
    "[data-magellan-expedition]" .sub-nav a, [data-magellan-expedition-clone] .sub-nav a {
      line-height: 1.8em; }
```
This is pretty bad, since this results in not working css. Any ideas / questions? :/

**Comment by xzyfer** [comment]
There are some known issues with interpolation in the new beta. Please open
an issue with LibSass.

**Comment by devshack** [comment]
But it appears in node-sass 3.4.2, too (not only in the beta).

**Comment by xzyfer** [comment]
Alrighty. Either way it's a bug in LibSass.

**Comment by devshack** [comment]
A bug in libsass 3.3.2, then (since it's working in 3.3.3)?

**Comment by xzyfer** [comment]
Both if it's still happening in the beta.

**Comment by devshack** [comment]
You're right: https://github.com/sass/libsass/issues/1770
It's supposed to be fixed in libsass 3.3.3 ... so, when does node-sass gets updated to it? ;)
